modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- An unexpected exception was thrown when switching to readonly mode and adjusting the editor width #TINY-6383
```

Comment:
Should we mention that this was specifically for a CSS resize?

Reply:
No, as it wasn't... you can potentially trigger it other ways as well (resizing the window for example, having something in the container change size as well, etc...).